Source/cmake/OptionsWPE.cmake

```diff
@@ -110,6 +110,12 @@ WEBKIT_OPTION_DEPEND(ENABLE_DOCUMENTATION ENABLE_INTROSPECTION)
 
 WEBKIT_OPTION_CONFLICT(USE_SOUP2 ENABLE_WPE_1_1_API)
 
+# Supported platforms.
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_WESTEROS "Whether to enable support for the Westeros platform" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_BCM_NEXUS "Whether to enable support for the Broadcom Nexus platform" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_RPI "Whether to enable support for the Raspberry Pi" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WESTEROS_SINK "Westeros-Sink to be used as video-sink for GStreamer video player" PUBLIC OFF)
```

Comment:
Are you sure you want these to be build options rather than runtime checks? It would seem pretty unfortunate for standard builds of WPE WebKit to not include the Raspberry Pi support, in particular, since those devices are very popular.

Reply:
Yes.